client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -1,11 +0,0 @@
-/** @format */
-
-/**
- * Internal dependencies
- */
-import './utils/public-path';
-import './utils/block-category'; // Register the Jetpack category
-import 'gutenberg/extensions/markdown/editor';
-//import 'gutenberg/extensions/publicize/editor';
-import 'gutenberg/extensions/related-posts/editor';
-import 'gutenberg/extensions/tiled-gallery/editor';
```

Comment:
This preset file is imported in Calypso: 

https://github.com/Automattic/wp-calypso/blob/5c5c85f1b6cc70a6bf853ae4c7decc54a42dd75e/client/gutenberg/editor/edit-post/editor.js#L20

You might want to either keep it (and perhaps rename) or import each extension separately in `client/gutenberg/editor/edit-post/editor.js`.